### DISCUSSION TITLE
thanhle.blog/tu-duy-toi-uu

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#90] [Ideas]
# thanhle.blog/tu-duy-toi-uu

https://thanhle.blog/blog/tu-duy-toi-uu

**Comment by hai123vn** [comment]
em vừa đi làm lại sau kì nghỉ tết và vô check blog anh liền có 2 bài đọc đỡ chán hahaha. Chúc anh 1 năm mới bội thu và khi nào offline vậy anh

**Reply by thanhlmm** [reply]
:))) offline fan hả

**Comment by hai123vn** [comment]
đúng rồi anh, là 1 cơ hội gặp gỡ trao đổi kinh nghiệp làm việc và cuộc sống
xoay quanh code nữa hê hê

Vào Th 5, 15 thg 2, 2024 vào lúc 08:59 Thanh Le ***@***.***>
đã viết:

**Comment by minhdoan99** [comment]
lần đầu biết tới blog của anh, cảm giác đầu tiên là thấy thật may mắn, sẽ theo dõi blog của anh thường xuyên.
